pom.xml

```diff
@@ -142,7 +142,7 @@
           <version>${jib-maven.plugin.version}</version>
           <configuration>
             <from>
-              <image>gcr.io/distroless/java11-debian11</image>
+              <image>eclipse-temurin:17.0.4.1_1-jre-alpine</image>
```

Comment:
This section needs to be added in the 2 others apps/`pom.xml` files in order to override the default value.
```
<from>
  <image>eclipse-temurin:17.0.4.1_1-jre-alpine</image>
</from>
```

Reply:
Is this not the root POM that gets used for all services?

Reply:
Oh ok gotcha, didn't see this root `pom.xml` properly, yep, all good then :)